nio_cli/commands/list.py

```diff
@@ -13,6 +13,10 @@ def run(self):
         response = requests.get(self._base_url.format(self._resource),
                                 auth=self._auth)
         try:
-            [print(resource) for resource in response.json()]
+            [print(value['id'], value['name']) for key,
+                                    value in response.json().items()]
         except:
-            pass
+            try:
+                [print(resource) for resource in response.json()]
```

Comment:
Nested `try/except` if id is not a value available. Attempts to print using the old method, otherwise fail silently